ci-operator/step-registry/cucushift/installer/rehearse/azure/upi/baselinecaps/v411/provision/cucushift-installer-rehearse-azure-upi-baselinecaps-v411-provision-chain.yaml

```diff
@@ -0,0 +1,18 @@
+chain:
+  as: cucushift-installer-rehearse-azure-upi-baselinecaps-v411-provision
+  steps:
+  - ref: ipi-install-rbac
+  - chain: ipi-conf-azure
```

Comment:
missing step `additional-enabled-capabilities`?

Reply:
baselineCapabilitySet is set to v4.11, it will install all optional operators in v4.11 set, no more additional operators to be installed, so no need to add step `additional-enabled-capabilities` in this chain.

Reply:
same as https://github.com/openshift/release/pull/30855/files#r931916863, ignore my comment